engine/apps/mobile_app/auth.py

```diff
@@ -43,9 +42,4 @@ def authenticate_credentials(self, token_string: str) -> Tuple[Optional[User], O
         except InvalidToken:
             return None, None
 
-        if auth_token.organization.is_moved:
```

Comment:
This will no longer be needed if the mobile app updates its backend URL based on the status endpoint.

Reply:
what about for older versions of the mobile app that don't do that?

Reply:
This was the incomplete first attempt at fixing the issue, so it was not really useful. Since we paused the migration of orgs with mobile users we will need make sure the mobile app version that uses this status endpoint is rolled out before we start migration again.